rotkehlchen/chain/evm/decoding/utils.py

```diff
@@ -102,7 +102,8 @@ def bridge_match_transfer(
     event.event_type = new_event_type
     event.event_subtype = HistoryEventSubType.BRIDGE
     event.counterparty = counterparty.identifier
+    bridge_suffix = '' if counterparty.label.endswith('bridge') else ' bridge'
```

Comment:
I would actually change this and make sure all counterparties passed here are consistent. Either they are all bridges or they are all the protocol without a bridge.

To do that remove the existence of xDAI Bridge as protocol. It's the old name anyway. Use `CPT_GNOSIS_CHAIN` as the protocol to pass for the bridge and let this be as it was.